src/parser/core/modules/CooldownDowntime.tsx

```diff
@@ -0,0 +1,264 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import {getDataBy} from 'data'
+import ACTIONS from 'data/ACTIONS'
+import {Action} from 'data/ACTIONS/ACTIONS'
+import {CastEvent} from 'fflogs'
+import Module, {dependency} from 'parser/core/Module'
+import {Requirement, Rule} from 'parser/core/modules/Checklist'
+import React from 'react'
+import Checklist from './Checklist'
+import Downtime from './Downtime'
+
+const DEBUG_EXPECTED_CALCULATION = false && process.env.NODE_ENV !== 'production'
+
+interface CooldownReset {
+	/**
+	 * One or more skills that trigger a cooldown reset.
+	 */
+	actions: Action[],
```

Comment:
I'm guessing this is for stuff like WAR's gauge spenders dropping cd on infuriate? Being an array seems _kinda_ weird to me, but i guess that works.